AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5018] SG2 | Update Build Config for Resource Tabs

## Ticket(s)

**Jira Ticket**
- [EWL-5018: SG2 | Code standards to-dos for Resource page](https://issues.ama-assn.org/browse/EWL-5018)

## Description
This update adds the tabs.js file to the array of drupalfiles in our build.config.

## To Test

- Pull `bugfix/EWL-5018-update-build-config` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Page > Resource](http://localhost:3000/?p=pages-resource).
- Confirm page still works.
- Open a local version of AMA One with assets pointed to local versions
-- Edit your local `provisioning/vars/local.yml` to use your local SG2 files
-- Run `vagrant up` and then `scripts/build` to update symlinks to css and js files.
- Confirm that tabs.js is loading.


## Visual Regressions
N/A


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
